packages/vm/src/types.ts

```diff
@@ -274,12 +274,30 @@ export interface RunBlockOpts {
    * Default: `false` (HF is set to whatever default HF is set by the {@link Common} instance)
    */
   setHardfork?: boolean | BigIntLike
+
+  /**
+   * If true, adds a hashedKey -> preimages mapping of all touched accounts
+   * to the `RunTxResult` returned.
+   */
+  reportPreImages?: boolean
 }
 
 /**
- * Result of {@link runBlock}
+ * Result of {@link applyBlock}
  */
-export interface RunBlockResult {
+export interface ApplyBlockResult {
```

Comment:
Hmm, I  think changing the name of this interface is breaking?

Reply:
It shows up weird in the git diff, but I just added an ApplyBlockResult interface, and modified RunBlockResult so that it extends it.